cypress/fixtures/vrn861.6718154672094133.mrc

```diff
@@ -0,0 +1 @@
+01108cam  2200313Ia 4500001001300000003000600013005001700019008004100036020001800077020001500095024001200110035002100122040005200143043001200195050002000207082001600227090002500243100002400268245008100292260005300373300002900426502008500455504005400540651005400594651004700648651003400695980003200729981003300761ocn768167901OCoLC20120411112059.3111207s2011    ne a     b    000 0 eng d  a9789077922903  a90779229038 a14567-1  a(OCoLC)768167901  aBTCTAbengcBTCTAdGUAdYDXCPdNhCcYBPdNhCcYBP  an-us-ga 4aF289b.B87 201104a975.802223  aF289b.B87 2011 ROCK1 aButler, Jay Jordan.10aAgrarianism and capitalism in early Georgia, 1732-1743 /cJay Jordan Butler.  aGroningen, the Netherlands :bBarkhuis,c[2011].  a145 p. :bill. ;c24 cm.  aOriginally produced as author's thesis (Masters) -- University of Wyoming, 1949.  aIncludes bibliographical references (p. 142-144). 0aGeorgiaxHistoryyColonial period, ca. 1600-1775. 0aGeorgiaxPolitics and governmentyTo 1775. 0aGeorgiaxEconomic conditions.  aKU/CC/DI/MbF289c.B87 2011  a1bxyzt12424527181891262626201216cam  22003734a 4500001001300000003000600013005001700019008004100036010001700077040003200094019001400126020003800140020003500178020001500213020001800228035003900246050002200285082001400307084001800321090003400339100002400373245007000397250001200467260004000479300004500519500005400564500002000618504005100638650003500689650003100724700002800755980003400783981002500817ocn320803622OCoLC20110104123129.8090526s2010    iauab         001 0 eng    a  2009021381  aDLCcDLCdBTCTAdCDCdDEBBG  a460040460  a9780072826845 (pbk. : alk. paper)  a0072826843 (pbk. : alk. paper)  a0070164592  a9780070164598  a(OCoLC)320803622z(OCoLC)46004046000aQE28.3b.D68 201000a551.7222  aRB 101232rvk  f1-SIZEaQE28.3b.D68 2010 SCI1 aProthero, Donald R.10aEvolution of the Earth /cDonald R. Prothero, Robert H. Dott, Jr.  a8th ed.  aDubuque, IA :bMcGraw-Hill,cc2010.  axxv, 518 p. :bcol. ill., maps ;c28 cm.  aDott's name appears first on the earlier edition.  aIncludes index.  aIncludes bibliographical references and index. 0aHistorical geologyvTextbooks.07aHistorische Geologie.2swd1 aDott, Robert H.,d1929-  aKU/CC/DI/MbQE28.3c.D68 2010  a1bszt97166227181891
```

Comment:
please add deleting this file to after block

Reply:
Done!